socorro/signature/tests/test_rules.py

```diff
@@ -1417,30 +1417,33 @@ def test_predicate(self):
     def test_action_success(self):
         rule = rules.SignatureIPCChannelError()
 
-        crash_data = {"ipc_channel_error": "ipc" * 50}
+        channel_error = "ipc" * 50
```

Comment:
`"ipc" * 50` creates a string that exceeds 100 characters, so this tests truncation of that part in the final signature.